### ISSUE TITLE
How to build doc?

### ISSUE THREAD

**Issue by devxpy** [nucleic/enaml#297]
```
$ git clone https://github.com/nucleic/enaml.git
$ cd enaml
$ cd docs
$ make html
Makefile:40: *** missing separator.  Stop.
```

Removed line 40 that had the `example:` clause...

```
$  make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.7.6

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/home/dev/.pyenv/versions/3.7.0/lib/python3.7/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/home/dev/.pyenv/versions/3.7.0/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 23, in <module>
    from enaml.version import version_info
ModuleNotFoundError: No module named 'enaml'

make: *** [Makefile:40: html] Error 2
```
```
$ python build.py gh_pages
M	docs/Makefile
Already on 'master'
Your branch is up to date with 'origin/master'.
From https://github.com/nucleic/enaml
 * branch              master     -> FETCH_HEAD
Already up to date.
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.7.6

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File "/home/dev/.pyenv/versions/3.7.0/lib/python3.7/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File "/home/dev/.pyenv/versions/3.7.0/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File "conf.py", line 23, in <module>
    from enaml.version import version_info
ModuleNotFoundError: No module named 'enaml'

make: *** [Makefile:40: html] Error 2
Traceback (most recent call last):
  File "build.py", line 81, in <module>
    handlers[arg]()
  File "build.py", line 50, in gh_pages
    shutil.copytree('./docs/build/html', temp_dir + '/docs')
  File "/home/dev/Projects/enaml/venv/lib/python3.7/shutil.py", line 309, in copytree
    names = os.listdir(src)
FileNotFoundError: [Errno 2] No such file or directory: './docs/build/html'
```

**Comment by MatthieuDartiailh** [comment]
Since I need to review a PR about improving the docs I will add a section about building them, I guess. From the top of my head, you need at least to install enaml, before building the docs and optionally install the dedicated pygment lexer found under tools/pygments (using `python setup.py install`) to get the example syntax highlighting.

**Comment by devxpy** [comment]
I wrote some myael

**Comment by MatthieuDartiailh** [comment]
This was fix in #294. Feel free to reopen your PR.
